### PR TITLE
fix: Some issues with onboarding for 3000

### PR DESCRIPTION
**Pull request by benjackwhite**
## Problem

Some remaining work to be done

## Changes

* Fixes the flex layout for the product cards
* Fixes a bunch of button sizes and issues
* Fixes the input selected to just use the border color

Screenshot 2023-11-20 at 17 20 31
Screenshot 2023-11-20 at 17 20 07

## TODO
- [x] Fix the topbar not being shown for 3000
- [ ] Confirm signup/login buttons look correct

👉 *Stay up-to-date with [PostHog coding conventions](https://posthog.com/docs/contribute/coding-conventions) for a smoother review.*

### CODE REVIEW

File: frontend/src/lib/lemon-ui/LemonInput/LemonInput.scss [hunk 1]
```diff
@@ -17,10 +17,18 @@
...
+        .posthog-3000 & {
+            border-color: var(--border-bold);
+        }
     }
 
     &.LemonInput--focused:not([aria-disabled='true']) {
         border-color: var(--primary-3000);
+
+        .posthog-3000 & {
+            border-color: var(--border-bold);
```
Comment (daibhin) [inline frontend/src/lib/lemon-ui/LemonInput/LemonInput.scss:30]: I assume this changes everywhere in 3000? Feels like a big change but understand the concerns about it looking like an error state. Just wondering if grey is the greatest active state

**Review by daibhin — APPROVED** [review]

File: frontend/src/lib/lemon-ui/LemonInput/LemonInput.scss (hunk 1 above)
Comment (benjackwhite) [reply frontend/src/lib/lemon-ui/LemonInput/LemonInput.scss:30]: I think it is miles better than orange (i.e. nearly red). So yeah I would vote for it as the default until we have a better option

**Comment by raquelmsmith** [comment]
Poked at this a bit. I have code for the following changes to the top nav but will wait until I get the go/no-go from y'all before I push to this branch:

1. Only show project switcher if the org has multiple projects
2. Make the user button tertiary
3. Make both buttons muted so the icon isn't red (not sure where we are on this from a 3000-colors perspective, but it felt really abrupt here)
4. Added a border below the minimal nav. I'm kinda iffy on this because generally our top "nav" in 3000 (i think mostly breadcrumbs) doesn't have a border, but it looked kind of naked without, and since in this view this is the "primary" nav, I think the border is okay.

Only one project:
image

Multiple projects:
image

**Comment by benjackwhite** [comment]
@raquelmsmith I put in the changes as in your screenshots 👍

**Comment by raquelmsmith** [comment]
awesome thanks so much!

**Comment by raquelmsmith** [comment]
Whoops, I think this broke onboarding - bridge pages are no longer scrollable. My fault really since I have no tests for onboarding flows.... I'm going to revert this since it's not super critical and then we can fix

**Pull request merged**